packages/web/src/common/store/lineup/sagas.js

```diff
@@ -73,6 +78,10 @@ function* filterDeletes(tracksMetadata, removeDeleted) {
         return null
       }
 
+      if (!allowedIds.includes(metadata.owner_id)) {
+        return null
```

Comment:
wrong this impact all lineups?

Reply:
good call whoops

Reply:
reopening comment per offline discussion to see if we can filter on lineup source so we don't block these tracks on all profiles